src/actors/rqgActorSheet.ts

```diff
@@ -1403,6 +1403,39 @@ export class RqgActorSheet extends ActorSheet<
         (createdItems[0] as RqgItem)?.sheet?.render(true);
       });
     });
+
```

Comment:
@wake42 these sections are the same or similar and could be generic if runes, skills, and passions all had the same interface or just the same properties.  I think this could be a good time to make a change for the future to support occupations and cultures:  (we could also finish this issue and do what I'm suggesting in another issue or as a part of the occupations and cultures changes)

.baseChance (for passions this would be 60, for skills it would vary)
.modifiers (maybe this is an array of key-value pairs?  Occupations and cultures and such will add permanent bonuses here)
.gainedChance (or maybe we could call it .experience or .experienceGain)
.categoryMod (only for skills, maybe even refactor this to just be another entry in modifiers)
.chance (the sum of the previous three)

Reply:
Not sure it makes sense to have a baseChance on runes & passions, especially not power runes that are matched against another rune.
The `data-rune-grid-edit` &. `data-passion-grid-edit` is the same thing though, I agree. For occupation / culture using this you mean there should be a modifier field for all these items that can be modified by those items, either by active effect or raw changing when you embed an occupation for example?
I guess we could implement a key-value structure and have a modifiers array, but arrays are not that easy to handle in Foundry dataModels I think. So if you wanted to link one of those to an input field it would take some extra code to accomplish.
Talking about datamodels that is also work that needs to be done - migrating the current definitions to use the datamodels Foundry uses: #596 
All in all I'm not sure it's good to squeeze all these items into the same mold - they are different I think (even if they are similar)